distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -699,12 +699,16 @@ private static Integer findPortNumber(Integer portToTry) {
     return new Option[0];
   }
 
-  private Option[] configureEmbeddedSolr() {
+  private Option[] configureSolr() {
+
     return options(
-        editConfigurationFilePut(SYSTEM_PROPERTIES_REL_PATH, "solr.client", "EmbeddedSolrServer"),
-        editConfigurationFilePut(SYSTEM_PROPERTIES_REL_PATH, "solr.http.url", ""),
+        editConfigurationFilePut(SYSTEM_PROPERTIES_REL_PATH, "solr.client", "HttpSolrClient"),
+        editConfigurationFilePut(
+            SYSTEM_PROPERTIES_REL_PATH, "solr.http.url", "http://localhost:9994/solr"),
```

Comment:
We cannot do dynamic port allocation because we are starting up the Solr instance before the tests start. So yes  it prohibits running multiple itests in parallel. 

Is parallel itest running being taken advantage of anywhere currently?

Reply:
It has value for our Windows CI, where we've seen ports not always get released cleanly by the underlying system.